modules/understand-admission-controller-enforcement.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * operating/use-admission-controller-enforcement.adoc
+:_module-type: CONCEPT
+[id="understand-admission-controller-enforcement_{context}"]
+= Understanding admission controller enforcement
+
+If you intend to use admission controller enforcement, consider the following:
+
+* *API latency*: Using admission controller enforcement increases Kubernetes  or {ocp} API latency because it involves additional API validation requests.
+Many standard Kubernetes libraries, such as fabric8, have short Kubernetes or {ocp} API timeouts by default.
+Also, consider API timeouts in any custom automation you might be using.
+* *Image scanning*: You can choose whether the admission controller scans images while reviewing requests by setting the *Contact Image Scanners* option in the cluster configuration panel.
+** If you enable this setting, {product-title} contacts image scanners if scan or image signature verification results are not already available, which adds considerable latency.
```

Comment:
"...contacts the image scanners if the scan or image signature..." Add "the" in two places.